package-examples/cassandra/statefulset.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: apps/v1
```

Comment:
Is this a good example for the intro? It contains quote a bit of yaml, most of which is pretty specific to Cassandra. Would it be better to start with a simpler example, maybe nginx?